pkg/config/config.go

```diff
@@ -97,8 +97,8 @@ type RTCConfig struct {
 	// allow TCP and TURN/TLS fallback
 	AllowTCPFallback *bool `yaml:"allow_tcp_fallback,omitempty"`
 
-	// check RTT for TCP fallback, will skip tcp ice if signaling RTT greater than tcpGoodRTT(80ms)
-	TCPFallbackCheckRTT *bool `yaml:"tcp_fallback_check_rtt,omitempty"`
+	// if set, will skip tcp ice if signaling RTT greater than the threshold
+	TCPFallbackRttThreshold int `yaml:"tcp_fallback_rtt_threshold,omitempty"`
```

Comment:
can we remove this from the config file for now? would prefer to not expose it to the end user just yet.